frontend/src/components/UserTables/UnscheduledTable.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import { Passenger } from '../../types/index';
+import './unscheduledTable.css';
+import RiderCard from '../RiderCard/RiderCard';
+
+
+function renderTableHeader() {
+  return (
+    <tr>
+      <th></th>
+      <th className="tableHeader">Time</th>
+      <th className="tableHeader">Passenger</th>
+      <th className="tableHeader">Pickup Location</th>
+      <th className="tableHeader">Dropoff Location</th>
+      <th className="tableHeader">Needs</th>
+    </tr>
+  );
+}
+
+const Table = () => {
+  const passengers = [
+    { startTime: '8:20am', endTime: '8:40am', name: 'Rose Lisborn', pickupLocation: 'Eddygate', pickupTag: 'Ctown', dropoffLocation: 'Hollister Hall', dropoffTag: 'West', needs: 'Crutches' },
+    { startTime: '8:30am', endTime: '8:50am', name: 'Rose Lisborn', pickupLocation: 'Eddygate', pickupTag: 'Ctown', dropoffLocation: 'Hollister Hall', dropoffTag: 'West', needs: 'Crutches' },
+    { startTime: '9:10am', endTime: '9:40am', name: 'Rose Lisborn', pickupLocation: 'Eddygate', pickupTag: 'Ctown', dropoffLocation: 'Hollister Hall', dropoffTag: 'West', needs: 'Crutches' },
+    { startTime: '9:30am', endTime: '9:50am', name: 'Rose Lisborn', pickupLocation: 'Eddygate', pickupTag: 'Ctown', dropoffLocation: 'Hollister Hall', dropoffTag: 'West', needs: 'Crutches' },
+    { startTime: '10:10am', endTime: '10:30am', name: 'Rose Lisborn', pickupLocation: 'Eddygate', pickupTag: 'Ctown', dropoffLocation: 'Hollister Hall', dropoffTag: 'West', needs: 'Crutches' },
```

Comment:
are you creating passengers as a type in the frontend for a ride's rider? if not you can find the rider type in the [backend docs](https://www.notion.so/Carriage-API-Doc-e78892c2e23e42259aa332b5b3136f22). also time in the database is either ISO 8601 formatted or 24 hour representation, i think youll be receiving the ISO 8601 time from the backend in this case so update the code to handle that instead

Reply:
Yup! Since there isn't an endpoint available to get the exact information needed to display, I just hardcoded everything. Once I make the endpoint, I will adjust.